_std/text.dm

```diff
@@ -35,6 +35,12 @@ var/global/regex/discord_emoji_regex = new(@{"(?:<|&lt;)(?:a)?:([-a-zA-Z0-9_]+):
 /proc/discord_emojify(text)
 	return discord_emoji_regex.Replace(text, {"<img src="https://cdn.discordapp.com/emojis/$2.gif" onerror="if (this.src != 'https://cdn.discordapp.com/emojis/$2.png') this.src = 'https://cdn.discordapp.com/emojis/$2.png';" title="$1" width="32" height="32">"})
 
+/proc/linkify(text)
+	. = replacetext(text, "http://", "")
+	. = replacetext(., "https://", "")
+	. = full_url_regex?.Replace(., "<a target='_blank' href='http://$0'>$0</a>")
```

Comment:
It might be a good idea to maybe rewrite this in a way that keeps `https` if there's already `https` there. Not sure about the cleanest way but one simple way is to do the replacement first and then replace `http://http` with `http`. :shelterfrog:
Also the `?.` is sorta unnecessary. I'd rather have this crash and produce a runtime if full_url_regex is null for some reason.

Reply:
Sure, I can look into keeping https://!

iirc I used ?. because some thing used ?. with the existent url_regex and I just assumed there was a reason for that, but I can get rid of it